src/crimson/common/formatter.h

```diff
@@ -9,5 +10,13 @@ namespace std {
 
 ostream& operator<<(ostream& out,
                     const seastar::lowres_system_clock::time_point& t);
+template <typename T>
+ostream& operator<<(ostream& out, std::vector<T> v) {
+  out << " vec{";
+  for (auto &val : v) {
+    out << val;
```

Comment:
Missing spaces/commas between elements. We should be able to re-use fmt's vanilla vector<T> formatter included in `#include <fmt/ranges.h>`.

Reply:
Done. Please take a look, again. Thanks:-)